packages/salesforcedx-vscode-core/test/unit/commands/forceOrgOpen.test.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
```

Comment:
This file was mistakenly left in, I will remove this from the PR. The test was moved to the integration/commands directory.